app/scripts/metamask-controller.js

```diff
@@ -714,12 +874,16 @@ module.exports = class MetamaskController extends EventEmitter {
   }
 
   /**
-   * Import (lost) Accounts
+   * Probably no longer needed, related to the Version 3 migration.
+   * Imports a hash of accounts to private keys into the vault.
    *
-   * @param  {Object} {lostAccounts} @Array accounts <{ address, privateKey }>
+   * Described in:
+   * https://medium.com/metamask/metamask-3-migration-guide-914b79533cdd
    *
    * Uses the array's private keys to create a new Simple Key Pair keychain
    * and add it to the keyring controller.
+   * @deprecated
+   * @param  {Object} {lostAccounts} @Array accounts <{ address, privateKey }>
```

Comment:
I'm not sure if `@Array accounts <{ address, privateKey }>` is valid description syntax.